zkevm-circuits/src/copy_circuit.rs

```diff
@@ -324,26 +326,81 @@ impl<F: Field> SubCircuitConfig<F> for CopyCircuitConfig<F> {
                 },
             );
 
-            // addr change: for tx log, addr use addr_slot as index, not increase by 1
+            // for all cases, rw_counter + rwc_inc_left keeps same
             cb.condition(
                 and::expr([
                     not_last_two_rows.expr(),
```

Comment:
Shouldn’t this be `not is_last` (instead of not last two) ?

Reply:
yeah, you are right.